cmake/Caffe2Config.cmake.in

```diff
@@ -85,7 +85,13 @@ if(@USE_CUDA@)
   # be found again when including the Caffe2 target.
   set(CAFFE2_USE_CUDA @USE_CUDA@)
   set(CAFFE2_USE_TENSORRT @USE_TENSORRT@)
+
+  # Add current directory to module path so we pick up FindCUDAToolkit.cmake
+  set(old_CMAKE_MODULE_PATH CMAKE_MODULE_PATH)
+  list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
   include("${CMAKE_CURRENT_LIST_DIR}/public/cuda.cmake")
+  set(CMAKE_MODULE_PATH old_CMAKE_MODULE_PATH)
```

Comment:
@peterbell10 This is assigning the string 'old_CMAKE_MODULE_PATH' to the 'CMAKE_MODULE_PATH'.
This is causing issues in downstream projects.

https://github.com/pytorch/audio/actions/runs/4287763310/jobs/7469002979#step:8:529